vertx-web-client/src/main/java/io/vertx/ext/web/client/HttpResponse.java

```diff
@@ -119,20 +120,29 @@
    */
   @CacheReturn
   @Nullable
-  String bodyAsString();
+  default String bodyAsString() {
```

Comment:
Why do we need the default impl here now? To simplify `HttpResponse` implementations in general? I guess it's because interceptors may replace the response object.